src/app/layout/header/header.component.scss-theme.scss

```diff
@@ -15,6 +15,29 @@
     }
   }
   .top-bar {
+    color: mat-color($primary, '500-contrast');
+    background-color: mat-color($primary);
+
+    .menu-container {
+      .row {
+        button {
+          &.active {
+            background-color: mat-color($background, grey);
+            color: mat-contrast($accent, 500);
+            border-radius: 0;
```

Comment:
the theme file should only contain color-related styles

Reply:
Ok. I will remove it.